scss/index.scss

```diff
@@ -13,3 +13,20 @@
 ts-wrapper:not(:defined) {
   visibility: visible;
 }
+
+// Make the wrapper's children to appear as if they were direct children of the wrapper's parent, ignoring the wrapper itself.
+ts-wrapper {
+  display: contents;
+}
+
+// EXPERIMENTAL: Rules that have to do with the order of the elements need to be passed along via a :has selector, and that rule needs to be removed from the child
+// Note this this only one of such exceptions. The others will need to be added as we find them.
+// TODO: Need to look at just pulling all or some classes off the top-level div in wrapped components and applying those classes to the wrapper
+// or not using the wrapper element at all, and applying the classes to the top level div of the VC directly
+ts-wrapper:has(.last\:border-b-0:last-child) {
```

Comment:
not sure if it's necessary to have lines `26-30`. When I removed `.last:border-b-0` from the `ts-wrapper`'s child it has the same result as lines `26-30` (displays the border and still includes the bottom border of the last child)

Reply:
yeah, the rule on 26 isn't quite working yet. I'd like to leave it in here as a starting point, though.